lib/error/index.js

```diff
@@ -73,7 +73,7 @@ MongooseError.Messages = MongooseError.messages;
  *
  * @api public
  * @memberOf Error
- * @static DocumentNotFoundError
+ * @static
```

Comment:
shouldnt we then add @name DocumentNotFoundError?

Reply:
it is already added by `jsdocparsecomments` / `dox` automatically, i dont quite know where it is fetching that information from, but it is there (probably the next line after the comment?)

also by jsdoc standard, `@static` does not have any parameters or really a description